haskell/nix.bzl

```diff
@@ -48,3 +48,119 @@ targets()
     nixpkgs_package(
         **nixpkgs_args
     )
+
+def _bundle_impl(repository_ctx):
+    build_file_content = """
+package(default_visibility = ["//visibility:public"])
+    """
+    for package in repository_ctx.attr.packages:
+        build_file_content += """
+alias(
+    name = "{package}",
+    actual = "@{base_repo}-{package}//:pkg",
+)
+        """.format(
+            package = package,
+            base_repo = repository_ctx.attr.base_repository,
+        )
+    repository_ctx.file("BUILD", build_file_content)
+
+bundle = repository_rule(
```

Comment:
Should have a docstring/`docs` attribute. I opened an issue upstream: https://github.com/bazelbuild/bazel/issues/6271, so we can reference that.